Makefile

```diff
@@ -132,7 +132,12 @@ ifeq ($(USE_MKLDNN), 1)
 		LDFLAGS += -L$(MKLROOT)/lib
 	endif
 	CFLAGS += -I$(MKLDNNROOT)/include
-	LDFLAGS += -L$(MKLDNNROOT)/lib -lmkldnn -Wl,-rpath,'$${ORIGIN}'
+	# MKLDNN but to needs to be dynamically linked for windows as not all VS compilers support static linking
+	ifneq ($(UNAME_S), Windows)
+		LIB_DEP += $(MKLDNNROOT)/lib/libmkldnn.a
+	else
+		LDFLAGS += -L$(MKLDNNROOT)/lib -lmkldnn -Wl,-rpath,'$${ORIGIN}'
```

Comment:
VS compiler doesn't know those flags. Do we support VS build with Makefile?

Reply:
Nope, windows uses cmake

Reply:
@marcoabreu Thanks for explanation. This is as I expected. So we don't need to consider Windows build in Makefile.

Reply:
Why don't remove `LDFLAGS += -L$(MKLDNNROOT)/lib -lmkldnn -Wl,-rpath,'$${ORIGIN}'`? It doesn't work on Windows.

Reply:
The line was there before. I am not trying to change builds on windows.

Reply:
This line is used for Linux MKLDNN dynamic build. Then why not remove it?